service/k8sclient/k8client.go

```diff
@@ -396,7 +445,19 @@ func (c *K8Client) UpdateXtraDBCluster(ctx context.Context, params *XtraDBParams
 }
 
 // DeleteXtraDBCluster deletes Percona XtraDB cluster with provided name.
-func (c *K8Client) DeleteXtraDBCluster(ctx context.Context, name string) error {
+func (c *K8sClient) DeleteXtraDBCluster(ctx context.Context, name string) error {
+	secret := &common.Secret{
+		TypeMeta: common.TypeMeta{
+			APIVersion: "v1",
+			Kind:       "Secret",
+		},
+		ObjectMeta: common.ObjectMeta{
+			Name: fmt.Sprintf(pxcSecretNameTmpl, name),
+		},
+	}
+
+	_ = c.kubeCtl.Delete(ctx, secret)
```

Comment:
Let's first remove db cluster and then remove secrets.
And let's log if we will have an error here.